azerothcore/azerothcore-wotlk
# fix(DB/Creature): Ribbon Pole Dance NPC's

Ribbon Pole Dance debug NPC's

##### CHANGES PROPOSED:

- Adding missing Debug NPC's for the Event mindsummer.
- Using 10 free guids from DB.  


###### ISSUES ADDRESSED:


Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2054


##### TESTS PERFORMED:

Yes, tested in game. Everything is fine, while Clicking on the Ribbon Pole, player spining around with a blue spell effect coming from the top of the pole to the player. Applyed the 10% exp buff to the player in +3 min increments for every couple of seconds of spinning. 


##### HOW TO TEST THE CHANGES:

1. Event start mindsummer
2. .go c 204949,204950,204951,204952,204953,204954,204955,204956,204957,204958
3. Click on Ribbon Pole
4. You will see this effect like on image below:

![pole effect](https://user-images.githubusercontent.com/41213210/60664466-4b5ed300-9e62-11e9-9d4a-7deea1cd9826.png)



##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
